cob_behaviour/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(cob_behaviour)
```

Comment:
I discussed it with @ipa-fmw and we decided add it to cob_robots, because we want to use a different script depending on the robot(cob4-X, cob3-X, raw3-X...). cob_command_tools should be generic and usable independently of the robot configuration.